inst/rmarkdown/templates/copernicus/resources/template.tex

```diff
@@ -111,11 +111,6 @@
 %$header-includes$
 %$endfor$
 
-$if(algorithms)$
-\usepackage{algorithmic}
-\usepackage{algorithm}
-$endif$
```

Comment:
Removing this and the associated yaml option will be a breaking change for all current users who sets it. 

Is it not part of the Copernicus template anymore ? 

I believe we can leave that in the template and suggest to set it to false by default in the YAML header ? 

Which would that not work ?

Reply:
Not necessarily, because the `copernicus.cls` imports them either way. It was basically doubled in the `*.tex`, therefore Copernicus sent the manuscript back and ask to remove it.

Reply:
Oh you mean it is included in the cls file by copernicus already ?

I did not check but yes it is ! 
https://github.com/rstudio/rticles/blob/4aa0a44e4f41b02cfb1d5c1cdc80043b5f42e502/inst/rmarkdown/templates/copernicus/skeleton/copernicus.cls#L1681-L1711

so that is enough if you have the package installed ! 

I don't think **tinytex** will automatically install them though if missing - which is different than when you use `\usepackage`

Reply:
Yes, I do agree that **tinytex** would be of no help here and cannot install missing packages automatically. On the other hand, the CLS uses also other (more commonly used; admittedly) packages. But it appears that the build process would just continue if those packages are missing in the available LaTeX installation.

Reply:
> But it appears that the build process would just continue if those packages are missing in the available LaTeX installation.

And it means it will fail if you use one of the command from those packages without having installing it ? 

I guess this is usual LaTeX user experience. We could prevent that if we have a way to know which are the packages expected, otherwise we can't really I think

Reply:
@cderv Just to make it clear to me: **tinytex** will not recognise the `\RequirePackage{..}` in the CLS file?